### DISCUSSION TITLE
Removing camera cause the UI a bank screen

### DISCUSSION THREAD

**Discussion by Bomberski** [blakeblackshear/frigate#11660] [Ask A Question] [answered]
### What is your question:

Using 0.14.0-b97e274

When removing camera after installing the Beta the GUI with show a blank screen, It might load for 1 Sec and then blank. using PC and Android will do the same thing. I will load my config from 13 and it will work as it should. But if i edit the config to remove the camera it will not load

**Comment by NickM-27** [comment]
What do you mean by removing camera

**Reply by Bomberski** [reply]
I just edited the config, removing the all the text from my printer camera
![Screenshot_20240531_095850_Samsung Internet](https://github.com/blakeblackshear/frigate/assets/51887282/0992beb2-186a-4c10-93e1-19de5ebd3aae)

**Reply by NickM-27** [reply]
sounds like the config is just not valid then

**Reply by Bomberski** [reply]
I can still see my cameras from home-assistant so it's running still just no UI

**Reply by Bomberski** [reply, marked as answer]
Figured it out, I left the camera in go2rtc. So if it's not removed in both places it will cause the ui not to load.
